components/clarinet-format/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "clarinet-format"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+clarinet-files = { path = "../clarinet-files", default-features = false, optional = true }
+clarity-repl = { path = "../clarity-repl" }
```

Comment:
We might not need `clarity-repl` here, but instead we can directy depend on `clarity`.
I'm not a big fan of `clarity-repl` re-exporting all of the VM, and it creates a dependency mess.

```toml
clarity = { workspace = true }
```

Eventually, you will also have to manage the clarity feature like for native and wasm (you can see this pattern in other Clarinet components, such as the repl

```toml
[features]
default = ["cli"]
cli = [
    "clarity/canonical",
    "clarity/developer-mode",
    "clarity/devtools",
    "clarity/log",
]
wasm = [
    "clarity/wasm",
    "clarity/developer-mode",
    "clarity/devtools",
]
```